llvm/test/CodeGen/ARM/tail-call-float.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
 ; RUN: llc -mtriple armv7 -target-abi aapcs -float-abi soft -O0 -o - < %s \
 ; RUN:   | FileCheck %s -check-prefix CHECK-SOFT -check-prefix CHECK
```

Comment:
Remove the `-check-prefix CHECK` if the test doesn't need it.

Reply:
Have removed